operations/workplace/people/working-at-mattermost/family-and-friends-day.md

```diff
@@ -0,0 +1,31 @@
+# Mattermost Family & Friends Day
+
+At Mattermost we care a lot about our communities: our customers, our open source contributors, our staff, and the friends and family that support our staff to do all that they do.
+
+Occasionally we’ll call a “Family and Friends Day”. Aligned with our time-off policies, this is a day to rest, relax, and recharge by spending time with the people in our personal lives who matter so much.
+
+We want to be especially thoughtful on the topic in the time of COVID, economic change, and changes to travel and academic environments.
+
+Our first “Family and Friends Day” will be on September 4th, 2020.
+
+We announced this program at [COM](https://handbook.mattermost.com/operations/operations/company-cadence#customer-obsession-meeting-aka-com), and will run the majority of the company with the minimal staff needed to meet our service level commitments - and we’d ask the staff who are on-call, or who choose to work that day, to take an alternate day off.
+
+We’ll publicly announce our #familyandfriendsday on Twitter, LinkedIn, and other social media, which will link to this notice page and our FAQ below, which can be referenced in out-of-office responders.
```

Comment:
```suggestion
Our #familyandfriendsday will be shared on Twitter, LinkedIn, and other social media, with a link to this notice page and FAQ. This can be referenced in out-of-office responders.
```